config/config.go

```diff
@@ -35,7 +35,7 @@ type Config struct {
 	}
 
 	Log struct {
-		Format      string `validate:"required"`
+		Format      string `validate:"required,oneof=json text"`
 		ForceColors bool
 		Level       string `validate:"required"`
```

Comment:
Should we do that for the log level too?

Reply:
No because logrus already have a function that check it and it is used in the `config.validate` function.

Reply:
So we could remove the `logrus.ParseLevel` test and just keep the test on the config like that we have consistent errors.

Reply:
So I will change the error format then to keep it consistent, ok for you? :)

Reply:
Or maybe not, the validation errors are more just not so good. 

`Key: 'Config.Log.Format' Error:Field validation for 'Format' failed on the 'oneof' tag`

I would rather change the validation error format to make it more human friendly.

Reply:
hum ok, just keep it like it is it's fine